src/pages/patterns/cards.mdx

```diff
@@ -0,0 +1,223 @@
+
+<Caption>
+These cards provide a short but comprehensive overview for each topic presented.
+</Caption>
+
+
+### When to use informational cards
+
+* Use to give equal hierarchy to all content items
+* Use to link users to important features, services or topics
+* Use to group multiple text-based content items together
+
+
+### When not to use informational cards
+
+* Do not use to group unrelated content or resources
+* Do not use to display lists of features
+* Do not use to accommodate long-form reading 
+
+
+## Resources
+
+Cards can be used to link to diverse resources such as learning material or media that support the main content or offering. Grouping resources together using cards provides a preview to each resource and allows a user to choose their own journey based on their needs.
+
+Examples of card components that can be used to showcase diverse resources:
+
+<Title>Card section — with images</Title>
+
+![Card link](../../images/pattern/cards/cards-resources.jpg)
+
+<Caption>
+Cards with images, when grouped together, provide a visual glimpse into the additional resources available.
+</Caption>
+
+
+### When to use resource cards
+
+* Use to present an array of resources in a single section of the page 
+* Use when users need to continue learning about a topic or offering
+* Use to group different content types and destinations together
+
+
+### When not to use informational cards
+
+* Do not use if a resource is specifically tied to content elsewhere on the page
+* Do not use when users are at the end of their pre-purchase journey
+
+
+## Use of icons
+
+Cards often include an icon that indicates the behavior or direction in which the user will take or the type of content available to the user after they click on a card.
+
+
+<Title>Icon usage</Title>
+
+* Use the `arrow—right` icon to indicate moving to a different page
+* Use the `arrow—down` icon to indicate jumping to a section of the same page
+* Use the `launch` icon to indicate opening a separate application or website
+* Use the `download` icon to indicate a file download
+* Use the `video` icon to indicate launching a video player
+
+
+<Title>Icon position</Title>
+
+There are two options to consider when positioning an icon on a card:
+
+1. bottom-left, 16px from the edge 
+
+![Card link](../../images/pattern/cards/cards-icons-1.jpg)
+
+This is the default and most common option, recommended by IBM.com Library. Having the icon in this position creates clear visual alignment with all text fields on the card.
+
+2. bottom-right, 16px from the edge 
+
+![Card link](../../images/pattern/cards/cards-icons-2.jpg)
+
+Less common but also effective, this option works best on single large cards such as the Feature card. 
+
+
+<Title>Icon color</Title>
+
+Card icons act as the primary indicator or call-to-action and therefore should use the color token `$interactive-01` when possible. In some cases, white arrows may be used on gray90 or gray100 cards.
```

Comment:
```suggestion
Card icons act as the primary indicator or call-to-action and therefore should use the color token `$interactive-04` when possible. In some cases, white arrows may be used on gray90 or gray100 cards.
```

We're using $interactive-04 (this one changes from blue-60 to blue-50) for accessibility on the dark theme.

Reply:
Trust @oliviaflory to know the correct token!